f/fluent-hc-4.1.x/fluent-hc-4.1.x_ubuntu_16.04.sh

```diff
@@ -0,0 +1,11 @@
+# Install dependencies
+apt update -y
+apt install git maven openjdk-8-jdk
+
+git clone https://github.com/dthadi3/fluent-hc-4.1.x.git
+
+cd fluent-hc-4.1.x/
+sed 's/1.5/1.8/g' pom.xml > tmp
+mv tmp pom.xml
+
+mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
What is the reason we are "skipping" the tests here ?

Reply:
There are no tests to run.

Reply:
If that is the case, you need not have to use "-DkipTests" right ?